server/flyway/sql/V51__add_user_terms_conditions_table.sql

```diff
@@ -0,0 +1,47 @@
+-- Create fam_user_terms_conditions table
+CREATE TABLE IF NOT EXISTS app_fam.fam_user_terms_conditions
+(
+    user_terms_conditions_id        bigint          GENERATED BY DEFAULT AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    user_id                         bigint          NOT NULL,
+    version_id                      int             NOT NULL,
+    create_user                     varchar(100)    NOT NULL,
+    create_date                     timestamp(6)    DEFAULT CURRENT_DATE NOT NULL,
+    update_user                     varchar(100),
+    update_date                     timestamp(6)    DEFAULT CURRENT_DATE
+);
+
+-- Add table/column comments
+COMMENT ON TABLE app_fam.fam_user_terms_conditions IS 'User Terms Conditions records identify the users who accept the terms and conditions, as well as the version of it.'
+;
+COMMENT ON COLUMN app_fam.fam_user_terms_conditions.user_terms_conditions_id IS 'Automatically generated key used to identify the uniqueness of a terms and conditions acceptance record.'
+;
+COMMENT ON COLUMN app_fam.fam_user_terms_conditions.user_id IS 'Unique ID to reference and identify the user within FAM system.'
+;
+COMMENT ON COLUMN app_fam.fam_user_terms_conditions.version_id IS 'Number to identity the version of the terms and conditions the user accepted.'
+;
+COMMENT ON COLUMN app_fam.fam_user_terms_conditions.create_user IS 'The user or proxy account that created the record.'
+;
+COMMENT ON COLUMN app_fam.fam_user_terms_conditions.create_date IS 'The date and time the record was created.'
+;
+COMMENT ON COLUMN app_fam.fam_user_terms_conditions.update_user IS 'The user or proxy account that created or last updated the record.'
+;
+COMMENT ON COLUMN app_fam.fam_user_terms_conditions.update_date IS 'The date and time the record was created or last updated.'
+;
+
+-- Create index
+CREATE INDEX ix_app_fam_fam_user_terms_conditions_user_id ON app_fam.fam_user_terms_conditions (user_id)
+;
+CREATE UNIQUE INDEX fam_tc_user_version_uk ON app_fam.fam_user_terms_conditions(user_id, version_id)
+;
+
+-- Add constraints
+ALTER TABLE app_fam.fam_user_terms_conditions ADD CONSTRAINT fam_user_terms_conditions_pk PRIMARY KEY (user_terms_conditions_id)
+;
+ALTER TABLE app_fam.fam_user_terms_conditions ADD CONSTRAINT Reffam_user_terms_conditions_user
+    FOREIGN KEY (user_id)
+    REFERENCES app_fam.fam_user(user_id)
+;
+
+-- -- Add permision on 'fam_user_terms_conditions' table for app access management api db user
+GRANT SELECT, INSERT ON app_fam.fam_user_terms_conditions TO ${api_db_username}
+;
```

Comment:
I'm really bad at naming, feel free to see if we want any better name, thanks so much!!